[Insights] Geo Map insights documentation added (#539)
wso2/docs-choreo-dev: en/docs/insights/view-api-insights.md

To download a pregenerated report, click **Download** for the relevant month and then click on the required format.

### Geo Map

> The **Geo Map** page allows you to see a geographical representation of your API usage. You can use this page to make business decisions such as identifying growing regions, analyzing the impact of a region specific marketing campaign, etc.

Reviewer: ```suggestion
The **Geo Map** page allows you to see a geographical representation of your API usage. You can use this page to make business decisions such as identifying growing regions, analyzing the impact of a region-specific marketing campaign, etc.
```

![Geo Map page](../assets/img/insights/geomap-page-full.png){.cInlineImage-full}

> To filter the information displayed on this page by API, select the required API(s) in the **API** field. **All** is selected by default.

Reviewer: ```suggestion
To filter the information displayed on this page by API, select the required API(s) in the **API** field. The default value is **All**.
```

Reviewer: Suggesting this change to avoid passive voice.

> This heat map highlights the countries with the appropriate color measured by relative color scale measured based on highest hit count to lowest non-zero hit count.

Reviewer: ```suggestion
This heat map highlights the countries with the appropriate color based on the hit count. It uses a relative color scale to depict the highest hit count to the lowest non-zero hit count.
```

![Geo Map Hit Scale](../assets/img/insights/geomap-hit-count-scale.png){.cInlineImage-half}

#### Viewing number of requests by country

> You can hover over the highlighted countries to view the country name and the total hit counts originated from that location. The map can be zoomed and panned using mouse gestures.

Reviewer: ```suggestion
You can hold the pointer over the highlighted countries to view the country name and the total number of hits from that location. You can also use the mouse to zoom the map or pan it.
```

![Geo Map by country](../assets/img/insights/geomap-requests-by-country.png){.cInlineImage-half}

!!! Note

>     Currently, this feature is only available for On Premise environments.

Reviewer: ``` suggestion
    Currently, this feature is only available for on-premise environments.

> Optionally the geo map data can be downloaded in a tabular format using the PDF or CSV download options using ![download](../assets/img/insights/insights-download-report.svg) on top right corner.

Reviewer: ```suggestion
Optionally you can download geo map data in a tabular format using the PDF or CSV download options by clicking ![download](../assets/img/insights/insights-download-report.svg) on the top right corner.
```
